Problems/2798/ans_2798.py

```diff
@@ -0,0 +1,19 @@
+#https://www.acmicpc.net/problem/2798
+
+def solve():
+    N, M = input().split()
+    Card = list(map(int, input().split()))
+    Max = 0
+    
+    #와 모르겠다...
+    for x in range(int(N)-2):
+        for y in range(x+1, int(N)-1):
+            for z in range(y+1,int(N)):
+                Sum = Card[x]+Card[y]+Card[z] 
+                if Sum <=int(M):
+                    if Sum >= Max:
```

Comment:
```suggestion
                if Sum <=int(M) and Sum >= Max:
```
두가지 조건이 모두 성립하면 되기 때문에 and로 조건을 추가하면 되지 않을까요?